kernel/kernel-benchmarks/src/test/java/io/delta/kernel/benchmarks/WorkloadOutputFormat.java

```diff
@@ -283,24 +294,36 @@ public String toString() {
   }
 
   @Override
-  public void iteration(BenchmarkParams benchParams, IterationParams params, int iteration) {}
+  public void iteration(BenchmarkParams benchParams, IterationParams params, int iteration) {
+    delegate.iteration(benchParams, params, iteration);
+  }
 
   @Override
   public void iterationResult(
-      BenchmarkParams benchParams, IterationParams params, int iteration, IterationResult data) {}
+      BenchmarkParams benchParams, IterationParams params, int iteration, IterationResult data) {
+    delegate.iterationResult(benchParams, params, iteration, data);
+  }
 
   @Override
-  public void startBenchmark(BenchmarkParams benchParams) {}
+  public void startBenchmark(BenchmarkParams benchParams) {
+    delegate.startBenchmark(benchParams);
+  }
 
   @Override
-  public void endBenchmark(BenchmarkResult result) {}
+  public void endBenchmark(BenchmarkResult result) {
+    delegate.endBenchmark(result);
+  }
 
   @Override
-  public void startRun() {}
+  public void startRun() {
+    delegate.startRun();
+  }
 
   @Override
   public void endRun(Collection<RunResult> result) {
-    System.out.println("End run results:");
+    // Note: We do not delegate endRun to the standard output format, as we are generating
```

Comment:
Just curious, instead of printing to stdout should we push the generated report to the delegate?